packages/web-components/src/components/va-checkbox/va-checkbox.scss

```diff
@@ -1,27 +1,57 @@
+@forward 'settings';
```

Comment:
Adding this because `usa-icon` module is required for the checkbox icon and the settings will map it to the correct icons folder.